examples/blaze/src/main/scala/com/example/http4s/blaze/demo/server/Module.scala

```diff
@@ -0,0 +1,65 @@
+package com.example.http4s.blaze.demo.server
+
+import cats.effect.Effect
+import cats.syntax.semigroupk._ // For <+>
```

Comment:
I kind of like just importing `cats.implicits._` so new people don't stumble.  I haven't found a lot of drawbacks.

Reply:
This I can't say I agree 100% with, mostly because `cats.implicits._` slows down editors like intellij bringing in loads of syntax classes.

Reply:
I found that the community is a bit divided on this. I think it's great using `cats.implicits._` when you know what you're doing (and as Jose says if your team is not using Idea) but for the sake of the examples I think qualified imports are a good way to teach people thinking about what they're actually bringing into scope.

Reply:
This, from the [Cats docs](https://typelevel.org/cats/typeclasses/imports.html), is why I soured on a la carte.

> Note: Beware that if you import a type class instance or its syntax twice, you will receive conflicting implicits with a less than helpful error message. This usually happens when importing different type classes in the same hierarchy or when importing syntax enrichment for all type classes using cats.syntax.all._ or cats.implicits._ together with a more specific import like cats.syntax.option._ or cats.instances.either._. 

Intermediate FP folks start to mix and match and then the compiler gives them no help.  It's still not the first thing I look for when someone asks for help, even though it's been the answer so many times.

Reply:
I've seen they even mix `cats.implicits._` with `cats.syntax` and that's even worse. On this one I stick with teaching them to understand what they're actually bringing into scope.

Reply:
>   think it's great using cats.implicits._ when you know what you're doing

fwiw, I find that it's the complete opposite. You can use specific imports if you know where things are. I've lost track how many teams my team had otherwise working code that wouldn't compile because of too many or too few imports.

I agree with 
> On this one I stick with teaching them to understand what they're actually bringing into scope.

But in the context of docs, you might not get such a chance

Reply:
Okay, so this is the last piece missing of this PR. I don't mind using `cats.implicits._` instead of `cats.syntax` if that's what the majority wants. You already know my point of view. Flip a coin :D

Reply:
I think we're just being pedantic.

IMO it's ok :+1: